### PR TITLE
Fix: Usr:Synchronize Jenkins job in the installation process

### PR DESCRIPTION
**Pull request by flopezag**
#### REVIEWERS

@pratid @jesuspg 
#### DESCRIPTION
- Synchronize the Jenkins job in the repository.
- Update Jenkins jobs on each execution of the jenkins setup
#### TESTING

Executed the curl from command line, not testing the execution of the changes of the jenkins setup.

### CODE REVIEW

**Comment by jesuspg** [comment]
LGTM

File: fiware-region-sanity-tests/resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml [hunk 1]
```diff
@@ -61,19 +63,32 @@ This job performs the Sanity Check on the region given as parameter:&#xd;
...
     <hudson.tasks.Shell>
-      <command># Execute &apos;exec&apos; script
-./$FIHEALTH_SANITY_PROJECT/resources/scripts/jenkins.sh exec
+      <command># Execute &amp;apos;exec&amp;apos; script
+./$FIHEALTH_SANITY_PROJECT/resources/scripts/jenkins.sh exec</command>
+    </hudson.tasks.Shell>
+    <hudson.tasks.Shell>
+      <command># WORKAROUND - Publish data into Master CB.
+
+MASTER_ADAPTER_URL=http://10.0.64.4:1337
```
Comment (pratid) [inline fiware-region-sanity-tests/resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml:72]: I think this URL should be parametrized and not explicitly included in job definition

File: fiware-region-sanity-tests/resources/scripts/jenkins.sh [hunk 2]
```diff
@@ -251,6 +251,12 @@ if [ -z "$FIHEALTH_ADAPTER_URL" -o -z "$FIHEALTH_CB_URL" ]; then
 	exit 3
 fi
 
+# Check FIHealth environment variables for Jenkins
+if [ -z "$JENKINS_USER" -o -z "$JENKINS_PASSWORD" ]; then
+	printf "Either Jenkins user or Jenkins password not specified\n" 1>&2
+	exit 3
+fi
```
Comment (pratid) [inline fiware-region-sanity-tests/resources/scripts/jenkins.sh:261]: As these new environment variables are compulsory, please include them at the initial comment of the script, together with a brief explanation.

File: fiware-region-sanity-tests/resources/scripts/jenkins.sh [hunk 3]
```diff
@@ -303,6 +309,18 @@ setup)
 	make
 	make install
 
+    # Update Restart job
```
Comment (pratid) [inline fiware-region-sanity-tests/resources/scripts/jenkins.sh]: Check indentation

File: fiware-region-sanity-tests/resources/scripts/jenkins.sh [hunk 4]
```diff
@@ -303,6 +309,18 @@ setup)
...
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-0-RestartTestServers/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-0-RestartTestServers.xml"
+
+    # Update SetUp job
```
Comment (pratid) [inline fiware-region-sanity-tests/resources/scripts/jenkins.sh]: Check indentation

File: fiware-region-sanity-tests/resources/scripts/jenkins.sh [hunk 5]
```diff
@@ -303,6 +309,18 @@ setup)
...
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-0-SetUp/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-0-SetUp.xml"
+
+	# Update Flow job
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-1-Flow/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-1-Flow.xml"
+
+	# Update Exec job
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-2-Exec-Region/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml"
+
```
Comment (pratid) [inline fiware-region-sanity-tests/resources/scripts/jenkins.sh:350]: I think we shouldn't put fixed URLs here, and use `$JENKINS_URL` instead. In fact, this is the enviroment variable we could check at the beginning of the script execution (and not `$JOB_URL`, which is unnecessary).

File: fiware-region-sanity-tests/resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml (hunk 1 above)
Comment (flopezag) [inline fiware-region-sanity-tests/resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml:72]: I suggest to leave it as it due to this is the script (workaround) to execute. It is just the definition of the variable to parametrize the curl operation.

**Comment by pratid** [comment]
LGTM

**Pull request merged**